index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Portfolio - Marcos Hernández</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="" />
+    <meta name="author" content="Marcos Hernández Campos" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <header class="d-flex container-md p-4 justify-content-between">
+      <span>Welcome</span>
+      <nav>
+        <button type="button" class="button">
+          <img src="icons/menu.svg" alt="Menu" />
+        </button>
+      </nav>
+    </header>
+
+    <section class="d-flex m-4 p-2 landing">
+      <div
+        class="d-flex flex-column align-content-between align-self-center container-sm py-2"
+      >
+        <h1 class="primary py-2 my-2">
+          Hey there, I'm Marcos,
+          <span class="secondary">I'm a software developer</span>
+        </h1>
+        <p class="py-2 my-2">
+          I can help you build a product, feature or website. Look through some
+          of my work and experience. If you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+
+        <div class="d-flex justify-content-evenly align-items-center my-2 p-2">
+          <button type="button" class="button" id="twitter">
+            <img src="icons/twitter.svg" alt="Twitter" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/linkedin.svg" alt="Linkedin" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/medium.svg" alt="Medium" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/github.svg" alt="Github" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/angellist.svg" alt="Angellist" />
+          </button>
+        </div>
```

Comment:
Please fix the social media icons as according to the requiremetns we are supposed to have :
The list of icons uses a `ul` tag to group them, and `li` tag per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
for example :point_down: 
```
<ul>
    <li> <a href="">social media icon</i></a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
</ul>
```

Reply:
@PinkMoon25 Thank you for the feedback. I've submitted the proper fixes.